test/ctr.bats

```diff
@@ -1497,3 +1497,20 @@ function teardown() {
 	echo "$output"
 	[ "$status" -eq 0 ]
 }
+
+@test "annotations passed through" {
+	start_crio
+	run crictl runp "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+	grep -E "\"owner\": \"hmeng\"" "$TESTDIR/crio-run/overlay-containers/$pod_id/userdata/config.json"
```

Comment:
To avoid the issue with the changing storage drivers, we could do something like:
```
> crictl inspectp $pod_id | grep -q '"owner": "hmeng",'
```

Reply:
Thanks! Should be good to go now